lib/pbench/server/cache_manager.py

```diff
@@ -786,6 +886,20 @@ def unpack(self, dataset_id: str) -> Tarball:
         tarball.controller.unpack(dataset_id)
         return tarball
 
+    def get_info(self, dataset_id: str, path: Path) -> dict:
```

Comment:
Using simply `dict` as a return type doesn't say much. And if there's much info here, using something like an `@dataclass` might be much better than just a `dict`.

Reply:
??

Reply:
I changed the return type to something more relevant.